packages/clarity-js/src/layout/schema.ts

```diff
@@ -39,7 +41,7 @@ export function ld(json: any): void {
                     dimension.log(Dimension.ProductCondition, json[JsonLD.ItemCondition]);
                     dimension.log(Dimension.ProductCurrency, json[JsonLD.PriceCurrency]);
                     dimension.log(Dimension.ProductSku, json[JsonLD.Sku]);
-                    metric.max(Metric.ProductPrice, num(json[JsonLD.Price]));
+                    metric.max(Metric.ProductPrice, num(json[JsonLD.Price], Setting.PriceScale));
```

Comment:
>PriceScale [](start = 84, length = 10)

Will require a server-side change?

Reply:
Rolled it back per our offline conversation.